timestamp.js

```diff
@@ -0,0 +1,42 @@
+const express = require("express");
+
+const app = express();
+
+app.set("view engine", "ejs");
+app.use(express.static("./static"));
+
+app.get("/timestamp/:timeStamp", (req, res) => {
+    const TimeStamp = req.params.timeStamp;
+
+    console.log("timeStamp : "+TimeStamp);
+    let timeStamp = "-";
+
+    if(TimeStamp === "") {
+        timeStamp = (new Date()).getTime();
+        utcString = (new Date()).toUTCString();
+    }
+    else if ( !isNaN(TimeStamp) ) {
+        timeStamp = (new Date(TimeStamp)).getTime();
+        utcString = (new Date(TimeStamp)).toUTCString();
+    }
+    else {
+        res.send(JSON.stringify({"error": "Invalid Date" }));
+    }
+    res.send(JSON.stringify({"date stamp": timeStamp, "utc" : utcString }));
+
+})
+
+app.get("/timestamp/", (req, res) => {
```

Comment:
did not understand what these two handlers do. Do you want to return error in case if the user is not hitting the right path?
```
app.get('*', function(req, res){
  res.status(404).send('what???');
});
```